app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,8 @@
  *= require_tree .
  *= require_self
  */
+
+@import "components/chart.scss";
```

Comment:
I would import gem components and then local components/styles, to be able to reuse variables or overwrite things if necessary.